vtex-sites/base.store
# feat: Remove postal code using query string

## What's the purpose of this pull request?
Remove the postal code component from being downloaded and rendered via query string.
⚠️ Please, give me better names for components and hooks.

## How does it work?
When the page URL has `__performanceTest` query string, the PreventLoadComponent prevents the PostalCodeInput from being loaded.

## How to test it?
Add the `__performanceTest` query string from the URL, and check that the browser does not fetch the PostalCode asset.

## References

## Checklist

<em>You may erase this after checking them all ;)</em>

- [] CHANGELOG entry added
